server/ngwmn/templates/site_location.html

```diff
@@ -225,6 +225,79 @@ <h1>{{ feature.SITE_NAME }}</h1>
                     </table>
                 </div>
             </li>
+            <li>
+                <button class="usa-accordion-button" aria-expanded="false" aria-controls="ts-a4">
+                    Water Level Statistics
+                </button>
+                <div id="ts-a4" class="usa-accordion-content">
+
+                    <table id="overallStats" class="usa-table-borderless waterStatsTable">
```

Comment:
Nit, but we're generally using dashes like `overall-stats` when dealing with CSS classes and node IDs.

Reply:
fixed in next push